packages/workflow/src/index.ts

```diff
@@ -11,6 +11,9 @@
  */
 // tslint:disable-next-line no-var-requires
 require('app-module-path').addPath(require('path').join(__dirname, '../'))
+require('dotenv').config({
```

Comment:
Wanted to use .env file to set LANGUAGES in my local environment for workflow :)